CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# CHANGELOG
+
+All notable changes to this project will be documented in this file.
+
+## [Version 1.0]
```

Comment:
Change the CHANGELOG format please. It includes date.

Reply:
Fixed